src/clients/organizationsService.js

```
@@ -0,0 +1,136 @@
+  INITIAL_RETRY_DELAY,
+} from '../constants.js';
+
+// Factory function to create an Organizations client
+export const createAwsOrganizationsClient = () => {
+  logger.debug('Creating AWS OrganizationsClient');
+  return new OrganizationsClient();
+};
+
+export class OrganizationsService {
+  constructor(organizationsClient, delayBetweenOperations = 15000) {
+    if (!organizationsClient) throw new Error('OrganizationsClient is required');
```

ESLint

[error] Expected { after 'if' condition.

(curly)